src/arch/riscv/pcstate.hh

```diff
@@ -62,7 +62,7 @@ class PCState : public GenericISA::UPCState<4>
 
     bool _compressed = false;
     RiscvType _rvType = RV64;
-    uint64_t _vlenb = VLENB;
+    uint64_t _vlenb = 256;
```

Comment:
Ditto

Reply:
This one should algo have default VLENB value, not VLEN as now. I am fixing it